main.cpp

```diff
@@ -40,9 +40,105 @@ send me a DM to check your pull request
 
  Wait for my code review.
  */
-
 #include <iostream>
-int main()
+using namespace std;
```

Comment:
Never use `using namespace`.  it's a hack and is the easiest way to create conflicts with your code when you use multiple frameworks in a project and each uses its own namespace.
- [ ] spell out `std::` when using stuff from the standard library.